### ISSUE TITLE
opening external links in new tab

### ISSUE THREAD

**Issue by tusharnankani** [inaxia/hashcode_web#10]
## Desciption and why is this needed to be done?

![image](https://user-images.githubusercontent.com/61280281/103516713-f6f14500-4e96-11eb-97bf-abb3c9bb5962.png)

The `repl.it` platform should open in a new tab because:
- the platform takes a few seconds to setup and create the environment.
- moreover, it is an external link, hence while it sets up, the user can browse through the website.

## Implementation:

placing `target="_blank"` in the `a` tag would do the job.

@Nishant2907

**Comment by Nishant2907** [comment]
Yes, that would be useful, although the button  where the list shows 3 options will be removed soon once we get a stable coding platform where the user can code without signing up. If you run the website and go to the coding platform, repl.it will ask the user to sign up. If you know any possible coding ground that doesn't require signing up of the users then u can send me the link to the coding ground. Once its reviewed, u can raise a PR for that.

**Comment by tusharnankani** [comment]
> If you know any possible coding ground that doesn't require signing up of the users then u can send me the link to the coding ground.

How about [snipp.in](https://github.com/haxzie/snipp.in)?

**Comment by Nishant2907** [comment]
Does it support the programming languages we have in the website? Kindly check and tell.

**Comment by tusharnankani** [comment]
> Does it support the programming languages we have in the website? Kindly check and tell.

@Nishant2907 It is an in-browser code editor that supports a bunch of languages. Note: It is an editor, and not a compiler.

If a compiler is needed, the best one would be [onlinegdb](https://www.onlinegdb.com/) that supports most of the languages. There are a bunch of free JavaScript editors and compilers without signing up, but that is language-specific.
